.circleci/config.yml

```diff
@@ -0,0 +1,202 @@
+          key: v1-tester-chrome-{{ checksum "package.json" }}
+          paths: 
+            - ~/node_modules
+      - run:
+          name: Lint source
+          command: npm run lint
+  test:
+    docker:
+      - image: circleci/node:6.14
+        environment:
+          MOCHA_FILE: $CIRCLE_TEST_REPORTS/junit/test-results.xml
+    steps:
+      - checkout
+      - restore_cache:
+          key: v1-tester-chrome-{{ checksum "package.json" }}
+      - run:
+          name: NPM install
+          command: npm install
+      - save_cache:
+          key: v1-tester-chrome-{{ checksum "package.json" }}
+          paths: 
+            - ~/node_modules
+      - run:
+          name: Run tests
+          command: npm run test:ci
+  merge_to_master:
+    docker:
+      - image: rainforestapp/circlemator:latest
+    steps:
+      - run:
+          name: Merge to master
+          command: circlemator self-merge --base-branch=master --compare-branch=develop
+  upload_staging:
+    docker:
+      - image: circleci/node:6.14
+        environment:
+          MOCHA_FILE: $CIRCLE_TEST_REPORTS/junit/test-results.xml
+    steps:
+      - checkout
+      - run:
+          name: Install awscli
+          command: sudo apt-get -y -qq install awscli
+      - restore_cache:
+          key: v1-tester-chrome-{{ checksum "package.json" }}
+      - run:
+          name: NPM install
+          command: npm install
+      - save_cache:
+          key: v1-tester-chrome-{{ checksum "package.json" }}
+          paths: 
+            - ~/node_modules
+      - run:
+          name: Build stage
+          command: ./deploy stg
+      - run:
+          name: Move extension manifest
+          command: mv staging_manifest.json extension/manifest.json
+      - run:
+          name: Zip extension 
+          command: cd extension && mkdir /tmp/extension && zip -r /tmp/extension/extension-$CIRCLE_SHA1.zip . -x *.git* -x circle.yml -x *.DS_Store*
+      - store_artifacts:
+          path: /tmp/extension
+  update_staging_sha:
+    docker:
+      - image: rainforestapp/heroku-deploy:latest
+    steps:
+      - checkout
+      - run:
+          name: Setup Heroku
+          command: .circleci/setup-heroku.sh
+      - run:
+          name: Update staging SHA
+          command: |
+            heroku config:set PLUGIN_VERSION=$CIRCLE_SHA1 -a schrute-stg
+  run_rainforest:
+    docker:
+      - image: rainforestapp/rainforest-cli:latest
+    steps:
+      - run:
+          name: Run Rainforest
+          command: 
+            rainforest-cli run --token "$rainforest_token" --environment-id 356 --folder 3982 --fg --conflict abort --description "$circle_project_reponame - $circle_branch $circle_build_num"
+  upload_prod:
+    docker:
+      - image: circleci/node:6.14
+        environment:
+          MOCHA_FILE: $CIRCLE_TEST_REPORTS/junit/test-results.xml
+    steps:
+      - checkout
+      - run:
+          name: Install awscli
+          command: sudo apt-get -y -qq install awscli
```

Comment:
I think there's a docker image for this?

Reply:
That's a possible improvement - we could compile JS in one container and then upload in another, but I don't want to spend more time on this at this moment